### ISSUE TITLE
Page anchor section titles should not be links

### ISSUE THREAD

**Issue by estelle** [mdn/yari#4554]
Page sectioning headers H2s are turned into links.

Yari turns 
`## Syntax`

into

```html
<h2 id="syntax">
<a href="#syntax" title="Permalink to Syntax">Syntax</a>
</h2>
```
it would be better if it was just:

```html
<h2 id="syntax">Syntax</h2>
```

They should just have the ID attribute. That is enough. The header doesn't need to link to itself.

**Comment by schalkneethling** [comment]
> They should just have the ID attribute. That is enough. The header doesn't need to link to itself.

I believe the reason for this was that it made it easier to get the permalink i.e.

1. Click the header
2. Grab the URL from the URL bar and share

@escattone @fiji-flo Thoughts?

**Comment by fiji-flo** [comment]
Also to just do a right click on a heading to copy a link to it. GitHub adds an anchor symbol just before the heading to do this.

**Comment by schalkneethling** [comment]
> Also to just do a right click on a heading to copy a link to it. GitHub adds an anchor symbol just before the heading to do this.

We used to have this, but there was performance issues. We had a fix, but it fell by the way side and was never resurrected. Perhaps we can bring it back to life as part of the redesign? @dalexsy?

**Comment by schalkneethling** [comment]
> Also to just do a right click on a heading to copy a link to it.

Haha! That is even easier 🤦

**Comment by dalexsy** [comment]
Is the question if we need an anchor icon or not? We still need a way to scroll to sections of the page because of the in-article navigation.

**Comment by escattone** [comment]
Yes, the points that @schalkneethling and @fiji-flo made above were my understanding as well, about why we were adding a link to the header.

**Comment by dalexsy** [comment]
I'd prefer if clicking on the header updated the URL without an icon. We'd always need to show an icon for touch users, and I think that'd be a bit much.
